streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/task/FlinkClusterWatcher.java

```diff
@@ -168,13 +135,29 @@ private void alert(FlinkCluster cluster, ClusterState state) {
    * @param flinkCluster
    * @return
    */
-  private ClusterState getClusterState(FlinkCluster flinkCluster) {
-    ClusterState state = getClusterStateFromFlinkAPI(flinkCluster);
-    if (ClusterState.isRunningState(state)) {
+  public ClusterState getClusterState(FlinkCluster flinkCluster) {
+    ClusterState state = FAILED_STATES.getIfPresent(flinkCluster.getId());
```

Comment:
I think there is a problem with the logic in this method: here, in Yarn Session mode, it won't go `jobmanagerUrl `request status, only `yarn restful api` requests will be processed. This is not in line with our original intention of designing the `jobmanagerUrl` field.